x-pack/plugins/endpoint/public/applications/endpoint/models/policy.ts

```diff
@@ -15,8 +15,17 @@ export const generatePolicy = (): PolicyConfig => {
   return {
     windows: {
       events: {
-        process: true,
+        api: true,
```

Comment:
@parkiino an updated list from @ferullo lists these as the supported Windows events on the first release

DLL/Driver Load
File
Network
Process
Registry
Security
DNS

I'll update the AC.  Apologies on the extra events in there, we can delete all that aren't in the list above

Reply:
gotcha i'll change it